cmake/llvm-version.txt

```diff
@@ -1 +1 @@
-8345289ded788f4df79f59df76df0c0437c3df64
+b96f18b20c31449ef9a6878b5c2725a7cf65c552
```

Comment:
Hi, could you help to update this to latest one, or at least covering my recent commit `f06563a5c0d239a6b98f74db522681613254ad08`. Thanks!

Reply:
updated.